rfcs/01929_tls_attestation.md

```diff
@@ -0,0 +1,258 @@
+# RFC#01929: Remote Attestation via TLS
+
+## Objective
+
+Remotely attest applications in Trusted Execution Environments (TEEs) over the
+TLS protocol.
```

Comment:
Please generalize the RFC to be about arbitrary assertions / claims, as discussed. TEE remote attestation may be one prominent example in the RFC, but the protocol must be more generic.

e.g. see https://tools.ietf.org/html/rfc5705#section-1 :

> A typical example is DTLS-SRTP [DTLS-SRTP], a key
   management scheme for the Secure Real-time Transport Protocol (SRTP)
   that uses DTLS to perform a key exchange and negotiate the SRTP
   [RFC3711] protection suite and then uses the DTLS master_secret to
   generate the SRTP keys.